src/documentation/markdown/GettingStarted/Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [5.18.1] - 2019-12-04
+### Fixed
+- Added a position absolute for the Tooltip component, which stopped working in Publish
```

Comment:
@MayaUribe, what do you think about removing the "which stopped working in Publish" part from the documentation? 🙂